From 6a2d214b55cb1e8517afccb18c3799b22e16a57e Mon Sep 17 00:00:00 2001
From: Daniel De Graaf <code@danieldg.net>
Date: Wed, 5 Aug 2020 20:31:57 -0400
Subject: [PATCH 1/2] Fix titles containing & and other HTML entities

---
 src/modules/wlr/taskbar.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/modules/wlr/taskbar.cpp b/src/modules/wlr/taskbar.cpp
index 23a91661..025e5c19 100644
--- a/src/modules/wlr/taskbar.cpp
+++ b/src/modules/wlr/taskbar.cpp
@@ -306,7 +306,7 @@ std::string Task::state_string(bool shortened) const
 
 void Task::handle_title(const char *title)
 {
-    title_ = title;
+    title_ = Glib::Markup::escape_text(title);
 }
 
 void Task::handle_app_id(const char *app_id)

From 4cd31cf3c36db71c04eabf3dad1f12b877991015 Mon Sep 17 00:00:00 2001
From: Daniel De Graaf <code@danieldg.net>
Date: Wed, 5 Aug 2020 20:35:41 -0400
Subject: [PATCH 2/2] Fix wlr/taskbar all-outputs config string

---
 src/modules/wlr/taskbar.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/modules/wlr/taskbar.cpp b/src/modules/wlr/taskbar.cpp
index 025e5c19..f7e11df9 100644
--- a/src/modules/wlr/taskbar.cpp
+++ b/src/modules/wlr/taskbar.cpp
@@ -709,7 +709,7 @@ bool Taskbar::show_output(struct wl_output *output) const
 
 bool Taskbar::all_outputs() const
 {
-    static bool result = config_["all_outputs"].isBool() ? config_["all_outputs"].asBool() : false;
+    static bool result = config_["all-outputs"].isBool() ? config_["all-outputs"].asBool() : false;
 
     return result;
 }
