From 3c182c9ca9dc8341388885ad0e1d0ed8b28c09aa Mon Sep 17 00:00:00 2001
From: Erik Reider <35975961+ErikReider@users.noreply.github.com>
Date: Mon, 23 May 2022 14:13:30 +0200
Subject: [PATCH] Added gamemode man file to meson

---
 meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/meson.build b/meson.build
index 21b89fe0..5a59df59 100644
--- a/meson.build
+++ b/meson.build
@@ -345,6 +345,7 @@ if scdoc.found()
         'waybar-cpu.5.scd',
         'waybar-custom.5.scd',
         'waybar-disk.5.scd',
+        'waybar-gamemode.5.scd',
         'waybar-idle-inhibitor.5.scd',
         'waybar-keyboard-state.5.scd',
         'waybar-memory.5.scd',
